src/index.ts

```diff
@@ -56,6 +56,12 @@ export interface UserSpec {
   id: number;
 }
 
+export interface SingleUserWorkspacePermissionSpec {
+  username: string;
+  workspace: string;
+  permission: string;
```

Comment:
Since `None` can be returned on the backend
```suggestion
  permission: string | null;
```